### DISCUSSION TITLE
Illegal offset type

### DISCUSSION THREAD

**Discussion by palexandrite** [statamic/cms#7729] [Troubleshooting] [answered]
I came across with the next error:

Illegal offset type

The message refer to vendor\laravel\framework\src\Illuminate\Translation\Translator.php: 126

It is popping up when I try to open any list of the collections page of CMS.

And my data of environment:
php 8.1.9
laravel 9.52.4
statamic 3.4.6

**Comment by jasonvarga** [comment, marked as answer]
Same as #7672 - have a look in there for a workaround.

What custom translation files have you created?

**Reply by palexandrite** [reply]
Thank you for the tip (link).

I have those:

![2023-03-21_11-08-20](https://user-images.githubusercontent.com/19309630/226515554-4e08d28e-d00c-4afa-9429-e635773a5bc6.png)

**Reply by palexandrite** [reply]
@jasonvarga yes, really thank you for the tip :-)
I was researching through my custom files of translation and found out that there was a conflict in translation keys.

Thank you pretty much one more :-)
